src/Infrastructure/Services/ContentfulService.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading.Tasks;
+using Application.Contracts;
+using Application.Responses.V1;
+using Contentful.Core;
+using Contentful.Core.Search;
+
+namespace Infrastructure.Services
+{
+    public class ContentfulService: IContentfulService
+    {
+        private readonly IContentfulClient _client;
+
+        public ContentfulService(IContentfulClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<ContentfulEmail> GetContentfulEmailAsync(string entryId, string locale = "en-GB")
```

Comment:
`CultureInfo locale` default can be handled with `null`.

`if(locale==null) { locale = new CultureInfo("en-GB"); }`